packages/admin-ui/src/pages/system/components/Repository/index.tsx

```diff
@@ -16,9 +16,23 @@ import {
 } from "services/dappnodeStatus/selectors";
 import { changeEthClientTarget } from "pages/system/actions";
 import Alert from "react-bootstrap/Alert";
-import { withToastNoThrow } from "components/toast/Toast";
+import { withToastNoThrow, withToast } from "components/toast/Toast";
+import { IpfsClient } from "components/IpfsClient";
+import { IpfsClientTarget } from "common";
+import SubTitle from "components/SubTitle";
 
 export default function Repository() {
```

Comment:
You should split this component into two independent ones, one for ipfs one for eth. Will keep the state, UI, etc together in one file.